Gemfile.lock

```diff
@@ -200,7 +200,7 @@ GEM
       gemoji (~> 3.0)
       html-pipeline (~> 2.2)
       jekyll (~> 3.0)
-    kramdown (1.17.0)
+    gem "kramdown", ">= 2.3.0"
```

Comment:
it needs attention because of the high security riscs towards the account password